aten/src/ATen/cpu/vec256/vec256_float.h

```diff
@@ -141,6 +141,9 @@ template <> class Vec256<float> {
   Vec256<float> sqrt() const {
     return _mm256_sqrt_ps(values);
   }
+  Vec256<float> rsqrt() const {
+    return _mm256_div_ps(_mm256_set1_ps(1), _mm256_sqrt_ps(values));
```

Comment:
Likewise, why cannot `_mm256_rsqrt_ps` be used?

Reply:
See other comment.